snuba/datasets/storages/transactions_ro.py

```diff
@@ -0,0 +1,26 @@
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas.tables import TableSchema
+from snuba.datasets.storage import ReadableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.events_common import (
+    all_columns,
+    mandatory_conditions,
+    query_processors,
+    query_splitters,
+)
+
+schema = TableSchema(
+    columns=all_columns,
+    local_table_name="sentry_local",
+    dist_table_name="sentry_dist_ro",
+    storage_set_key=StorageSetKey.TRANSACTIONS_RO,
+    mandatory_conditions=mandatory_conditions,
+)
+
+storage = ReadableTableStorage(
+    storage_key=StorageKey.TRANSACTIONS_RO,
+    storage_set_key=StorageSetKey.TRANSACTIONS_RO,
+    schema=schema,
+    query_processors=query_processors,
+    query_splitters=query_splitters,
+)
```

Comment:
These are not the right parameters for the transactions storage.
They need to match those in `storages/transactions.py`.
These seem to match those in errors_ro.